Stripe/STPCard.m

```diff
@@ -135,6 +137,54 @@ - (BOOL)isEqualToCard:(STPCard *)other {
     return [self.cardId isEqualToString:other.cardId];
 }
 
+#pragma mark - Description
+
+- (NSString *)description {
+    NSString *fundingDescription;
+
+    switch (self.funding) {
```

Comment:
I'll make the changes to align with the current pattern and we can think about it more later.